rbd/mirror_group.go

```diff
@@ -0,0 +1,480 @@
+package rbd
+
+/*
+#cgo LDFLAGS: -lrbd
+#include <errno.h>
+#include <stdlib.h>
+#include <rados/librados.h>
+#include <rbd/librbd.h>
+
+// Types and constants are copied from librbd.h with added "_" as prefix. This
```

Comment:
You do not need to copy these enums and structs if you guarantee they are included in librbd.h.

The `rbd_mirror_group_enable_dlsym` and all other `*_dlsym` functions can be removed as well.

This `import "C"` block probably only needs to include the `#cgo` and `#include` lines.

Reply:
Ahh, right.
Updated the code.